apps/coordinator/src/components/ScriptExplorer/SignatureImporter.jsx

```diff
@@ -398,21 +391,8 @@ class SignatureImporter extends React.Component {
 
         let publicKey;
         try {
-          const sequence = enableRBF ? 0xfffffffd : 0xffffffff;
```

Comment:
this was causing issues because we don't actually know what the sequence is and the signature commits to this value. This really should jut come from the psbt.